recipe/meta.yaml

```
@@ -27,19 +27,26 @@ requirements:
     - python
 
 test:
+  source_files:
+    - tests
   imports:
     - mmh3
   requires:
     - pip
+    - pytest
   commands:
     - pip check
+    - pytest tests
 
 about:
   home: https://github.com/hajimes/mmh3
```

I think the homepage has changed, check `pyproject.toml`

```suggestion
  home: https://pypi.org/project/mmh3/
```

---

Umm, technically, the real home is https://github.com/hajimes/mmh3.

---

@JeanChristopheMorinPerso how do we decide on the homepage?  The `pyproject.toml` points to PyPI (where the Homepage points to itself), the Github page also uses its documentation link (top right) to point to PyPI.

---

My logic is usually: website > source (github, etc) > PyPI as a last resort.

---

For reference, here are the top home_url (by domain name) in all the packages we maintain (not feedstock, packages) that have more than 50 hits:
```
                  home_url                  | total 
--------------------------------------------+-------
 github.com                                 |  5850
 CRAN.R-project.org                         |  4484
 x.org                                      |   186
 gnu.org                                    |   146
 docs.ropensci.org                          |   126
 developer.nvidia.com                       |   115
 r-project.org                              |   113
 bitbucket.org                              |    81
 gcc.gnu.org                                |    79
 airflow.apache.org                         |    78
 r-forge.r-project.org                      |    68
 gitlab.com                                 |    64
 pypi.org                                   |    64
 cran.r-project.org                         |    62
 llvm.org                                   |    61
 dirk.eddelbuettel.com                      |    55
```

---

This seems like an excellent policy question that we should note and discuss in our upcoming team meetings.